src/System.Memory/tests/ReadOnlySpan/SequenceCompareTo.byte.cs

```diff
@@ -71,10 +71,32 @@ public static void LengthMismatchSequenceCompareTo_Byte()
             Assert.True(result > 0);
         }
 
+        [Fact]
+        public static void SequenceCompareToEqual_Byte()
+        {
+            for (int length = 1; length < 128; length++)
+            {
+                var first = new byte[length];
+                var second = new byte[length];
+                for (int i = 0; i < length; i++)
+                {
+                    first[i] = second[i] = (byte)(i + 1);
+                }
+
+                var firstSpan = new ReadOnlySpan<byte>(first);
+                var secondSpan = new ReadOnlySpan<byte>(second);
+                int result = firstSpan.SequenceCompareTo<byte>(secondSpan);
+                Assert.True(result == 0);
```

Comment:
Wouldn't Assert.Equal be better?
```C#
Assert.Equal(0, firstSpan.SequenceCompareTo<byte>(secondSpan));
```
Same below.

Reply:
Done